src/Chapter2.hs

```diff
@@ -741,7 +759,7 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains x l = filter (\y -> elem x y) l
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to this function and lambda function as well  🙂 
But your implementation is already great 💯